sql/analyzer/rules.go

```diff
@@ -20,6 +20,7 @@ var DefaultRules = []Rule{
 	{"move_join_conds_to_filter", moveJoinConditionsToFilter},
 	{"eval_filter", evalFilter},
 	{"optimize_distinct", optimizeDistinct},
+	{"remove_unnecessary_converts", removeUnnecessaryConverts},
```

Comment:
What do you think about moving this rule to `OnceAfterDefault`? Is it necessary run it several times?

Reply:
@mcarmonaa It should be executed several times because the plan will change and could contain new unnecessary casts again.